examples/methods/plot_kmm_reweighting.py

```diff
@@ -0,0 +1,60 @@
+"""
+Kernel Mean Matching
+====================
+This example illustrates the use of KMM method [1] to correct covariate-shift.
+.. [1] J. Huang, A. Gretton, K. Borgwardt, B. Schölkopf and A. J. Smola.
+       'Correcting sample selection bias by unlabeled data.'
+       In NIPS, 2007.
+"""
+
+# Author: Antoine de Mathelin
+#
+# License: MIT License
+# sphinx_gallery_thumbnail_number = 1
+
+# %%
+import numpy as np
+import matplotlib.pyplot as plt
+from skada import KMMAdapter
+
+# %%
+# Generate sample bias dataset
+# ----------------------------
+np.random.seed(123)
+Xs = np.random.randn(100, 2) * .5
+Xt = np.random.random((20, 2))
+
+X = np.concatenate((Xs, Xt))
+sample_domain = np.ones(X.shape[0])
+sample_domain[Xs.shape[0]:] *= -2
```

Comment:
Usually we use the `make_shifted_datasets()` method  to generate our example datasets. Here you can use it with the arg `shift="covariate_shift"`